data-types.md

```diff
@@ -70,18 +69,15 @@ The list of allowed `vehicle_type` values in MDS. Aligning with [GBFS vehicle ty
 
 ## Vehicle Status 
 
-A vehicle status record represents the current or last-known disposition of a vehicle, defined as follows:
+A vehicle status record represents the current or last-known event and telemetry from a vehicle, defined as follows:
 
 | Field | Type | Required/Optional | Comments |
-| ----- | ---- | ----------------- | ----- |
+| ----- | ---- | ----------------- | -------- |
 | `device_id` | UUID | Required | A unique device ID in UUID format, should match this device in Provider |
 | `provider_id` | UUID | Required | A UUID for the Provider, unique within MDS. See MDS [provider list](/providers.csv). |
 | `data_provider_id` | UUID | Optional | If different than `provider_id`, a UUID for the data solution provider managing the data feed in this endpoint. See MDS [provider list](/providers.csv) which includes both service operators and data solution providers. |
-| `last_vehicle_state` | Enum | Required | [Vehicle state][vehicle-states] of most recent state change. |
```

Comment:
Should last state be removed? Otherwise if someone is needing this how can it be determined solely by last_event?

Reply:
see update